functions/concepts/src/main/java/com/example/functions/concepts/LazyFields.java

```diff
@@ -30,20 +30,26 @@
   // Always initialized (at cold-start)
   // Warning: Class variables used in Servlet classes must be thread-safe,
   // or else might introduce race conditions in your code.
-  private static final int nonLazyGlobal = fileWideComputation();
+  private static final int NON_LAZY_GLOBAL = fileWideComputation();
```

Comment:
nit: This says "fileWideComputation", but it's really "perStartupComputation". It's not done once per file, it's done once per class initialization.